cpp/models/ode_secirvvs/parameters.h

```diff
@@ -119,6 +119,38 @@ struct TestAndTraceCapacity {
     }
 };
 
+/**
+ * @brief Multiplier for the test and trace capacity to determine when it is considered overloaded from symptomatic cases.
+ */
+template <typename FP = double>
+struct TestAndTraceCapacityMaxRiskSymptoms {
+    using Type = UncertainValue<FP>;
+    static Type get_default(AgeGroup)
+    {
+        return Type(15.0);
+    }
+    static std::string name()
+    {
+        return "TestAndTraceCapacityMaxRiskSymptoms";
+    }
+};
+
+/**
+ * @brief Multiplier for the test and trace capacity to determine when it is considered overloaded from cases without symptoms.
+ */
+template <typename FP = double>
+struct TestAndTraceCapacityMaxRiskNoSymptoms {
```

Comment:
It would be a little more intuitive if the order of the parameter definitions was the same as in the definition of ParametersBase. But of course this is correct.